mite/logoutput.py

```diff
@@ -103,6 +111,17 @@ def process_message(self, message):
         elif msg_type in ("error", "exception"):
             self._error_total += 1
             self._error_recent += 1
+            # get the name of the erroring journey
+            journey_name = message.get("journey")
+            if journey_name is not None:
+                # the dictionary stores how many times
+                # the journey has errored
+                journey_error = self._error_journeys.get(journey_name)
+                if journey_error is None:
+                    self._error_journeys[journey_name] = 1
+                else:
+                    journey_error += 1
+                    self._error_journeys[journey_name] = journey_error
```

Comment:
```suggestion
                self._error_journeys[journey_name] += 1
```

This is enabled by the usage of `defaultdict`.  We probably want to make this code conditional on the value of `self._journey_logging` (so that we're not building a dict we don't use, in case journey_logging is False)